### PR TITLE
XW-22 don't call method getWiki on ForeignAPIRepo instance - such method doesn't exist

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

**Comment by bognix** [comment]
@hakubo says LGTM

**Comment by vforge** [comment]
LGTM

File: includes/wikia/services/WikiaFileHelper.class.php [hunk 1]
```diff
@@ -592,15 +592,23 @@ public static function getWikiaFilename( $url ) {
...
-			$info = VideoInfo::newFromTitle( $file->getTitle()->getDBkey() );
-			if ( empty( $info ) ) {
-				$isAdded = false;
+		if ( $file instanceof File && !$file->isLocal() ) {
+			$repo = $file->getRepo();
+			/**
+			 * When repo is an instance of ForeignAPIRepo
+			 * file comes from mediawiki and isn't store in any of wikias.
+			 */
+			if ( $repo instanceof ForeignAPIRepo ) {
```
Comment (Warkot) [inline includes/wikia/services/WikiaFileHelper.class.php:601]: Could it be written this way?

```
if (
    $repo instanceof ForeignAPIRepo
    || F::app()->wg->WikiaVideoRepoDBName == $repo->getWiki()
    && empty( VideoInfo::newFromTitle( $file->getTitle()->getDBkey() ) )
) {
    return false;
}
```

I am just wondering where to put the logical operators — end or start of the line.

File: includes/wikia/services/WikiaFileHelper.class.php [hunk 2]
```diff
@@ -592,15 +592,23 @@ public static function getWikiaFilename( $url ) {
...
 	public static function isAdded( $file ) {
-		$isAdded = true;
-		if ( $file instanceof File && !$file->isLocal()
-			&& F::app()->wg->WikiaVideoRepoDBName == $file->getRepo()->getWiki() ) {
-			$info = VideoInfo::newFromTitle( $file->getTitle()->getDBkey() );
-			if ( empty( $info ) ) {
-				$isAdded = false;
+		if ( $file instanceof File && !$file->isLocal() ) {
+			$repo = $file->getRepo();
+			/**
```
Comment (Warkot) [inline includes/wikia/services/WikiaFileHelper.class.php:597]: This is just a comment, not a doc-block, so maybe it would be better to use just the `//`?

**Pull request merged**